app/templates/home-view.jade

```diff
@@ -2,11 +2,18 @@ extends /templates/base-flat.jade
 
 block content
   .container-fluid
-    .row#covid19-banner
+    .row#top-banner
       .row
         .col-xs-12
-          span(data-i18n="new_home.covid19_banner_blurb")
-          a(href="/parents", data-i18n="new_home.learn_more_caps", data-event-action="Click: School Closure Banner")
+          - var bannerSwapDate = new Date('2020-06-29T06:00:00.000-07:00') // 6/2/9/20 6am PST
+          if new Date() > bannerSwapDate
+            span(data-i18n="new_home.top_banner_blurb")
+            a(href="https://www.google.com/url?q=https://www.seedinvest.com/codecombat?utm_campaign%3Dseedinvestcampaign%26utm_medium%3Dlandingpage%26utm_source%3Dcodecombat&sa=D&ust=1593124239826000&usg=AFQjCNHoJpwHDCPaVR6kbKDkPq1-8lJXDQ", data-event-action="Click: SeedInvest Banner")
```

Comment:
Is it intentional for this to be a google redirect url?

Reply:
good catch, this is a copy/paste error.  i had copied link instead of copied text from a google doc comment..  will update.